tox_conda/plugin.py

```diff
@@ -158,15 +163,26 @@ def tox_testenv_create(venv, action):
         # conda env create does not have a --channel argument nor does it take
         # dependencies specifications (e.g., python=3.8). These must all be specified
         # in the conda-env.yml file
+        yaml = YAML()
+        env_file = yaml.load(Path(venv.envconfig.conda_env))
+        env_file["dependencies"].append(python)
```

Comment:
I think that any already existing python in this list should be removed. From (bad) experiences, I tend to not trust the conda dependencies resolver.

Reply:
The disadvantage of removing the existing python dependency is that conflicts, such as a dependency python>=3.8 in the env file for tox env `py37` aren't caught, which would again be inconsistent with the `conda_deps` behavior, which fails if you specify `conda_deps= python>=3.8` in tox env `py37`.

I haven't run into any issues with the dependency resolver during my testing, but I can still remove the existing dependency from the env if you prefer that.

Reply:
I'm not used to have loosely defined environment but that makes sense.

Could you please tell a bit about this in the conda_env section of the readme?